### PR TITLE
[EN] Glossary

### PR DESCRIPTION
**Pull request by MegaApplePi**

### CODE REVIEW

File: wiki/Glossary/en.md [hunk 1]
```diff
@@ -536,16 +466,9 @@ IRC is an abbreviation for _Internet Relay Chat_.
 
 For more information about IRC, see [Internet Relay Chat](/wiki/Internet_Relay_Chat/).
 
-## J
-
-### Jump
+### Interface
```
Comment (TPGPL) [inline wiki/Glossary/en.md]: add link to article

File: wiki/Glossary/en.md [hunk 2]
```diff
@@ -967,26 +890,22 @@ However, if you have an [osu!supporter](/wiki/osu!supporter/) tag, the beatmapse
...
 ### Stacked
 
 Stacked objects are hit objects that overlap each other in the playfield.
 The most common object that is stacked are hit circles.
 
 Stacking hit circles (a clear violation of beat spacing) is, nevertheless, permitted if the hit circles are very close to each other in the timeline (half a beat or usually even less).
 
+### Standard
+
+_See also: [osu!standard](/wiki/osu!standard)._
```
Comment (TPGPL) [inline wiki/Glossary/en.md]: /wiki/osu!standard -> /wiki/Game_Modes/osu! or is it a redirect

Comment (MegaApplePi) [inline wiki/Glossary/en.md]: I was like 10 seconds lazy... :s but yeah this is a redirect-- I will write specs on it after #36 is done

File: wiki/Beatmaps/en.md [hunk 3]
```diff
@@ -1,3 +1,28 @@
...
+
+_Not to be confused with Beatmapsets_
+
+Beatmap packs, also known as _packs_, are sets of beatmaps packaged in a compressed file (usually `.zip`).
+Beatmap packs are collections of ranked and/or approved beatmaps that can be downloaded from a file sharing service (on [MediaFire](https://www.mediafire.com/)).
+
+Most beatmap packs are sets of beatmaps that were ranked during the same time period.
+Although, some packs are tailored to a similar style or theme.
+
+To see the beatmap packs, see [Beatmap Pack Listing](https://osu.ppy.sh/p/packlist).
```
Comment (sr229) [inline wiki/Beatmaps/en.md]: L21: Revise, Suggested ``[..] see the [Beatmap Pack listing] [..]``

File: wiki/Beatmaps/en.md [hunk 4]
```diff
@@ -1,3 +1,28 @@
...
+These could be related to those in the stages in the DS games.
+In their most basic form, beatmaps have a song and hit objects placed based on the rhythm of the music.
+
+Nowadays, beatmaps usually have a background image, video, or even a storyboard.
+
+For more details about how beatmaps are uploaded, see BSS.
+
+### Beatmap Pack
+
+_Not to be confused with Beatmapsets_
```
Comment (kj415j45) [inline wiki/Beatmaps/en.md]: > `_Do not confuse it with Beatmapsets_`

Comment (MegaApplePi) [inline wiki/Beatmaps/en.md]: the common convention for disambiguation is to use
```
_Not to be confused with {item}._
```
or at least, that is what Wikipedia uses...

File: wiki/Beatmaps/en.md [hunk 5]
```diff
@@ -1,3 +1,28 @@
+### Beatmaps
```
Comment (MegaApplePi) [inline wiki/Beatmaps/en.md]: this was a quick snippet dump so I don't forget to put it here, I will not fix it in this pr

File: wiki/Glossary/en.md [hunk 6]
```diff
@@ -967,26 +892,22 @@ However, if you have an [osu!supporter](/wiki/osu!supporter/) tag, the beatmapse
...
 ### Stacked
 
 Stacked objects are hit objects that overlap each other in the playfield.
 The most common object that is stacked are hit circles.
 
 Stacking hit circles (a clear violation of beat spacing) is, nevertheless, permitted if the hit circles are very close to each other in the timeline (half a beat or usually even less).
 
+### Standard
+
+_See also: [osu!standard](/wiki/Game_Modes/osu!)._
```
Comment (TPGPL) [inline wiki/Glossary/en.md]: See also: [osu!]?

File: wiki/Glossary/en.md [hunk 7]
```diff
@@ -967,26 +892,22 @@ However, if you have an [osu!supporter](/wiki/osu!supporter/) tag, the beatmapse
...
+
+Standard refers to the game mode in _osu!_ and has been used since it is the first game mode.
+This is sometimes stylized as _osu!standard_.
```
Comment (TPGPL) [inline wiki/Glossary/en.md]: as _osu!standard_ or osu!.

File: wiki/Glossary/en.md [hunk 8]
```diff
@@ -967,26 +892,22 @@ However, if you have an [osu!supporter](/wiki/osu!supporter/) tag, the beatmapse
...
+
+This term is used greatly inside the osu!wiki to prevent ambiguity between osu! (the framework, game client, etc.) and the game mode itself.
```
Comment (TPGPL) [inline wiki/Glossary/en.md]: italic osu! ?

File: wiki/Glossary/en.md [hunk 9]
```diff
@@ -1180,18 +1096,18 @@ Version has four meanings:
 
 ### WIP
 
-WIP is an acronym for work in progress.
+WIP is an acronym for _work in progress_.
```
Comment (TPGPL) [inline wiki/Glossary/en.md]: **W**ork **I**n **P**rogress

Comment (MegaApplePi) [inline wiki/Glossary/en.md]: bolding on osu-web is hard to differentiate when it is just single characters

File: wiki/Glossary/en.md [hunk 10]
```diff
@@ -364,6 +286,12 @@ _osu!_ currently uses two different algorithms to calculate difficulty stars:
 You can see a beatmap's difficulty when testing it in the editor.
 The difficulty stars can be seen on the song selection screen or at its listing in the site.
 
+### Downbeat
+
+The downbeat is a musical term for the first beat of a measure.
```
Comment (oralekin) [inline wiki/Glossary/en.md]: `Downbeat is a musical term`

File: wiki/Glossary/en.md [hunk 11]
```diff
@@ -406,8 +334,9 @@ The osu!standard game mode was is mostly based off of those games.
 
 EBA has two meanings:
 
-- Elite Beat Agents, one of the DS games which _osu!_ is based on
-- the collective term for the playable characters in that game (referring to meaning above), i.e. "The EBA" or "The Agents"
+- Elite Beat Agents, one of the DS games which _osu!_ is based on.
```
Comment (oralekin) [inline wiki/Glossary/en.md]: DS games **that** _osu!_ is based on

File: wiki/Glossary/en.md [hunk 12]
```diff
@@ -777,7 +647,7 @@ Ouendan has three meanings:
 - In Japanese culture, a cheer squad which, similar to western cheerleaders, rallies a sports team on using drums, horns, banners and yells of encouragement.
 - The pair of Japanese DS games that _osu!_ was based on.
```
Comment (oralekin) [inline wiki/Glossary/en.md]: that _osu!_ **is** based on

File: wiki/Glossary/en.md [hunk 13]
```diff
@@ -853,7 +715,7 @@ _osu!_ currently has three rankings:
   Scores achieved from ranked beatmaps are taken into account.
```
Comment (oralekin) [inline wiki/Glossary/en.md]: ln 704: `is usually the goal of beatmappers` or something, needs a reword imo

This line: Does anyone even check ranked score leaderboards anymore

File: wiki/Glossary/en.md (hunk 9 above)
Comment (oralekin) [inline wiki/Glossary/en.md]: I think he meant to capitalise the first letters...

File: wiki/Glossary/en.md (hunk 13 above)
Comment (MegaApplePi) [inline wiki/Glossary/en.md]: what to fix on line 715?

**Pull request merged**